### PR TITLE
Release v0.8.0

### PR DESCRIPTION
**Pull request by wongma7**
**Is this a bug fix or adding new feature?** new release

**What is this PR about? / Why do we need it?**
gp3 feature alone deserves a release.

I didn't bother to prune the changelog, I think i would prefer if it is exhaustive (i.e. contains ALL PRs) but just surface new features/bug fixes to top and keep everything else under misc.

**What testing is done?** TBH I am not sure how long it will take for the k8s.gcr.io image to becsome available since it is my first time doing it so switching the charts/docs to refer to it might be a bit premature. I can't leave them pointing to docker hub either because I still can't push v0.8.0 debian-based image to amazon docker hub, will only push the v0.8.0-amazonlinux one there.

### CODE REVIEW

File: charts/aws-ebs-csi-driver/templates/statefulset.yaml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-        {{- include "aws-ebs-csi-driver.labels" . | nindent 8 }}
-    spec:
-      serviceAccountName: ebs-snapshot-controller
-      containers:
-        - name: snapshot-controller
-          image: quay.io/k8scsi/snapshot-controller:v2.1.1
-          args:
-            - --v=5
-            - --leader-election=false
-{{- end }}
```
Comment (wongma7) [inline charts/aws-ebs-csi-driver/templates/statefulset.yaml:30]: I fixed line endings here and ran the new-ish `make generate-kustomize`

File: deploy/kubernetes/base/controller.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ spec:
         - operator: Exists
       containers:
         - name: ebs-plugin
-          image: amazon/aws-ebs-csi-driver:latest
+          image: k8s.gcr.io/aws-ebs-csi-driver:latest
```
Comment (wongma7) [inline deploy/kubernetes/base/controller.yaml:31]: actually this should be k8s.gcr.io/provider-aws/aws-ebs-csi-driver

**Comment by wongma7** [comment]
Also since the chart releaser action doesn't have permissions to update gh-pages index I will manually update it with chart-releaser later

File: CHANGELOG-0.x.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
+# v0.8.0
+
+## Notable changes
+- Images will be built on a Debian base by default. Images built on Amazon Linux will still be available but with the tag suffix `-amazonlinux`.
```
Comment (ayberk) [inline CHANGELOG-0.x.md:4]: Let's definitely add `gp3 is now default` here.

File: docs/README.md [hunk 4]
```diff
@@ -49,6 +50,7 @@ Following sections are Kubernetes specific. If you are Kubernetes user, use foll
 | AWS EBS CSI Driver \ Kubernetes Version| v1.12 | v1.13 | v1.14 | v1.15 | v1.16 | v1.17 | v1.18 |
```
Comment (ayberk) [inline docs/README.md:50]: Time to make this 1.18+? :)

File: CHANGELOG-0.x.md (hunk 3 above)
Comment (wongma7) [reply CHANGELOG-0.x.md:4]: done

File: docs/README.md (hunk 4 above)
Comment (wongma7) [reply docs/README.md:50]: yeah.

i was considering csompressing the rows and columns but I think tehre is some value in acknowleding each version explicitly...to a point. we might be past that point

**Comment by ayberk** [comment]
Let me summon the bot 🧙 
/lgtm

**Pull request merged**